addons/hr_holidays/models/hr_leave.py

```diff
@@ -199,8 +199,11 @@ def _default_get_request_dates(self, values):
     attachment_ids = fields.One2many('ir.attachment', 'res_id', string="Attachments")
     # To display in form view
     supported_attachment_ids = fields.Many2many(
-        'ir.attachment', string="Attach File", compute='_compute_supported_attachment_ids',
-        inverse='_inverse_supported_attachment_ids')
+        'ir.attachment', string="Attach File",
+        compute='_compute_supported_attachment_ids',
+        inverse='_inverse_supported_attachment_ids',
+        bypass_search_access=True,
```

Comment:
Wait, that field is non-stored and non-searchable. What's the point of `bypass_search_access`, then?